test/nodes/conftest.py

```
@@ -0,0 +1,462 @@
+# ---------------------------------------------------------------------------
+
+
+@pytest.fixture(autouse=True)
+def _reset_warnings():
+    """Clear captured warnings before each test."""
+    _warned_messages.clear()
+    yield
+    _warned_messages.clear()
+
+
+@pytest.fixture()
```

Ruff (0.15.7)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)